macros/SpecName.ejs

```diff
@@ -467,10 +467,6 @@ var specList = {
         name : 'Document Object Model (DOM) Level 3 XPath Specification',
         url  : 'https://www.w3.org/TR/DOM-Level-3-XPath/'
     },
-    'DOM4': {
```

Comment:
Several pages gave me "Sorry! We weren't able to save your contribution.":

- https://developer.mozilla.org/en-US/docs/Web/API/Document/createDocumentFragment
- https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/dispatchEvent
- https://developer.mozilla.org/en-US/docs/Web/API/Event/isTrusted